google-cloud-bigtable/lib/google/cloud/bigtable/chunk_reader.rb

```diff
@@ -0,0 +1,195 @@
+# frozen_string_literal: true
+
+module Google
+  module Cloud
+    module Bigtable
+      # Invalid row state error
+      class InvalidRowStateError < Google::Cloud::Error
```

Comment:
You might consider moving the custom error classes to `lib/google/cloud/bigtable/errors.rb`.

Reply:
Ok, I will move it into a separate file.